src/app/migration/src/src/app/modules/common/components/announcement-inbox-card/announcement-inbox-card.component.html

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Format the html and follow for all other html templates also

Reply:
done